plugins/course-apps/edxnotes/Settings.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from 'CourseAuthoring/pages-and-resources/app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+/**
+ * Settings widget for the "edxnotes" Course App.
+ * @param {{onClose: () => void}} props 
+ * @returns 
+ */
+const NotesSettings = ({ onClose }) => {
+  const intl = useIntl();
+  return (
+    <AppSettingsModal
+      appId="edxnotes"
+      title={intl.formatMessage(messages.heading)}
+      enableAppHelp={intl.formatMessage(messages.enableNotesHelp)}
+      enableAppLabel={intl.formatMessage(messages.enableNotesLabel)}
+      learnMoreText={intl.formatMessage(messages.enableNotesLink)}
+      onClose={onClose}
+    />
+    );
+  };
+  
+  export default NotesSettings;
```

Comment:
Commentary: This file is mostly unchanged (just moved) but since I changed `NotesSettings` to use `useIntl` instead of `injectIntl`, the indentation all changed and git shows it in the diff as a new file. Even though `useIntl` is way nicer than `injectIntl`, I didn't change the rest of these plugins, in order to make the diff simpler.